library/global.ml

```diff
@@ -209,10 +183,3 @@ let register_inline c = globalize0 (Safe_typing.register_inline c)
 
 let set_strategy k l =
   GlobalSafeEnv.set_safe_env (Safe_typing.set_strategy (safe_env ()) k l)
```

Comment:
En passant (not introduced by this PR): `set_strategy` should use `globalize0`, shouldn't it?